### PR TITLE
Fix database in preparation of deploy

### LINKED ISSUE #84: Only create database if it does not exist (by default)
set WIPE_AND_CREATE_DATABASE to false by default and create logic that checks for existing database and creates it if it doesn't exist

Also add bootstrap logic for generating password instead of just hardcoding it

### LINKED ISSUE #87: Create admin user
Set admin password with (Docker?) Environemnt variable

### PR DESCRIPTION
**Pull request by tobias-**
* Upgrade kotlin 1.2.20 -> 1.2.21
* Add admin role
* Warn about poor encryption keys
* Randomize passwords on first startup when creating database
* Don't default to wiping the database on startup

Fixes #87
Fixes #84
Fixes #103

### CODE REVIEW

File: tachikoma-database/src/main/kotlin/com/sourceforgery/tachikoma/config/DatabaseConfig.kt [hunk 1]
```diff
@@ -7,4 +7,5 @@ interface DatabaseConfig {
     val sqlUrl: URI
     val timeDatabaseQueries: Boolean
     val wipeAndCreateDatabase: Boolean
```
Comment (robin-wallberg) [inline tachikoma-database/src/main/kotlin/com/sourceforgery/tachikoma/config/DatabaseConfig.kt]: wipeOnCreate

File: tachikoma-database/src/main/kotlin/com/sourceforgery/tachikoma/database/hooks/CreateUsers.kt [hunk 2]
```diff
@@ -17,13 +20,21 @@ private constructor(
...
-            createBackendAuthentication(ebeanServer, account)
-            createFrontendAuthentication(ebeanServer, account)
-            createIncomingEmail(ebeanServer, account)
+        if (databaseConfig.createDatabase) {
+            ebeanServer.find(AccountDBO::class.java)
+                    .where()
+                    .eq("mailDomain", MAIL_DOMAIN)
+                    .findOne()
+            ?: let {
+
```
Comment (robin-wallberg) [inline tachikoma-database/src/main/kotlin/com/sourceforgery/tachikoma/database/hooks/CreateUsers.kt]: Plz remove

File: tachikoma-database/src/main/kotlin/com/sourceforgery/tachikoma/database/hooks/CreateUsers.kt [hunk 3]
```diff
@@ -17,13 +20,21 @@ private constructor(
...
     override fun postStart(ebeanServer: EbeanServer) {
-        if (databaseConfig.wipeAndCreateDatabase) {
-            val account = AccountDBO(MAIL_DOMAIN)
-            ebeanServer.save(account)
-            mqManager.setupAccount(MAIL_DOMAIN)
-            createBackendAuthentication(ebeanServer, account)
-            createFrontendAuthentication(ebeanServer, account)
-            createIncomingEmail(ebeanServer, account)
+        if (databaseConfig.createDatabase) {
+            ebeanServer.find(AccountDBO::class.java)
```
Comment (robin-wallberg) [inline tachikoma-database/src/main/kotlin/com/sourceforgery/tachikoma/database/hooks/CreateUsers.kt]: Check DB version

File: tachikoma-database/src/main/kotlin/com/sourceforgery/tachikoma/config/DatabaseConfig.kt (hunk 1 above)
Comment (tobias-) [reply tachikoma-database/src/main/kotlin/com/sourceforgery/tachikoma/config/DatabaseConfig.kt]: removed

File: tachikoma-database/src/main/kotlin/com/sourceforgery/tachikoma/database/hooks/CreateUsers.kt (hunk 3 above)
Comment (tobias-) [reply tachikoma-database/src/main/kotlin/com/sourceforgery/tachikoma/database/hooks/CreateUsers.kt]: Done

File: tachikoma-database/src/main/kotlin/com/sourceforgery/tachikoma/database/hooks/CreateUsers.kt (hunk 2 above)
Comment (tobias-) [reply tachikoma-database/src/main/kotlin/com/sourceforgery/tachikoma/database/hooks/CreateUsers.kt]: Done

**Pull request merged**